lib/timerage/time_interval.rb

```diff
@@ -59,5 +59,17 @@ def time_enumerator(step)
         end
       end
     end
+
+    # class methods
+    # =============
+
+    # Returns a new TimeInterval representation of the iso8601 interval
```

Comment:
You should document supported interval specifications. Right now this will only support `<start>/<end>`, not `<start>/<duration>`, `<duration>/<end>`, nor `<duration>`.